clusterloader2/pkg/measurement/common/wait_for_controlled_pods.go

```diff
@@ -192,14 +191,15 @@ func (w *waitForControlledPodsRunningMeasurement) gather(syncTimeout time.Durati
 	unknowStatusErrList := errors.NewErrorList()
 	timedOutObjects := []string{}
 	for _, checker := range w.checkerMap {
-		status, err := checker.getStatus()
+		objChecker := (checker).(*objectChecker)
```

Comment:
nit: checker.(*objectChecker)

[remove the () around checker]

Reply:
Done.